toolkit/scss/_temporary-message.scss

```diff
@@ -7,18 +7,14 @@
 // The following placeholders must be present in your main stylesheet:
 // - shared_placeholders/_temporary-messages.scss
 
-%temporary-message-section {
-  @extend %temporary-message;
-  margin-bottom: $gutter;
-}
-
+%temporary-message-section,
 .temporary-message {
-  @extend %temporary-message-section;
+  @extend %temporary-message;
```

Comment:
I mean, do you think it's worth adding an `@import "shared_placeholders/_temporary-messages.scss";` statement at the top of this file?